claster.py

```diff
@@ -17,31 +17,30 @@
         Covid_death.append(int(row['COVID-19 Deaths']))
         State.append(row['State'])
 
-States = set(State)
+States = list(set(State))
+state_to_num = {States[i]: i for i in range(len(States))}
 Covid_deaths = [0]*len(States)
 Deaths = [0]*len(States)
-i = 0
 
-for states in States:
-    for state in State:
-        if(state == states):
-            Covid_deaths[i] += Covid_death[i]
-            Deaths[i] += Death[i]
-    i += 1
+for i in range(len(State)):
+    j = state_to_num[State[i]] # index of seen State in actual states
+    Covid_deaths[j] += Covid_death[i]
+    Deaths[j] += Death[i]
```

Comment:
Works for O(len(State)) time.
Utilizes dictionary for judging to which "state death" value current death value should be added.